book/accessibility.md

```diff
@@ -883,7 +894,7 @@ class Tab:
 
 Finally, we just need to focus on the chosen element. If we've reached
 the last the focusable node (or if there weren't any focusable nodes
-to begin with), we'll unfocus the page and move focus to the address
+to begin with), we'll UN-focus the page and move focus to the address
```

Comment:
Would prefer italics rather than capitalization for emphasis here.

Reply:
That was a typo - changed to "un-focus"